src/material-experimental/theming/_m3-tokens.scss

```diff
@@ -384,6 +384,22 @@
   ));
 }
 
+/// Gets the token values for the MDC form field.
+/// @param {Map} $systems The MDC system tokens.
+/// @return {Map} The form field tokens.
+@function _get-form-field-tokens($systems) {
```

Comment:
should we take in the exclude-hardcoded param as well? the other functions here all take it

Reply:
I'm not quite sure what that parameter even does. I think I mostly saw it in MDC's tokens for opacities.